src/addons/__tests__/Grid.spec.js

```diff
@@ -626,7 +626,7 @@ describe('Grid', function() {
     });
 
     it('should generate the width based on the container size', function() {
```

Comment:
should add another test where we explicitly set the size as % and px and validate these are obeyed